bgpd/bgpd.h

```diff
@@ -1442,6 +1582,8 @@ struct bgp_nlri {
 /* BGP graceful restart  */
 #define BGP_DEFAULT_RESTART_TIME               120
 #define BGP_DEFAULT_STALEPATH_TIME             360
+#define BGP_DEFAULT_SELECT_DEFERRAL_TIME       360
+#define BGP_DEFAULT_RIB_STALE_TIME             500
```

Comment:
Actually the RIB stale timer value should logically be more than selection deferral time. However it depends on the operator configuration.
Since default selection deferral time is 360 so DEFAULT_RIB_STALE_TIME is chosen more than that i.e. 500